benchmarks-api/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Plz move this file out of api to examples module and call it `log4j2-example.xml`. Also we have to put some documentation  about this logging approach and share it on wiki page.